core/domain/blog_domain_test.py

```diff
@@ -453,6 +565,13 @@ def test_tags_validation_for_blog_post(self):
         self._assert_valid_tags_for_blog_post(
             'Tag should not be empty.', ['abc', ''])
 
-    def test_tags_validation_in_strict_mode(self):
+    def test_tags_validation_in_strict_mode(self) -> None:
         self._assert_strict_valid_tags_for_blog_post(
             'Atleast one tag should be selected', [])
+
+    def test_blog_post_tags_passes_validation(self) -> None:
+        """Tests validation for blog post tags."""
```

Comment:
You don't need to add docstrings for test functions.

Reply:
ditto elsewhere

Reply:
Should I remove all the docstring that were created before? Or just the ones made by me?

Reply:
Done